Sources_v3/StreamChat/Database/DTOs/MessageDTO.swift

```diff
@@ -302,6 +312,21 @@ extension _ChatMessage {
         localState = dto.localMessageState
         
         isFlaggedByCurrentUser = dto.flaggedBy != nil
+        
+        latestReactions = Set(
+            dto
+                .reactions
+                .sorted(by: { $0.updatedAt > $1.updatedAt })
+                .prefix(.maxNumberOfLatestReactions)
+                .map { $0.asModel() }
+        )
```

Comment:
I think it's better to use a fetch request for this (like we do it for replies). If there are thousands of reactions it makes no sense to load them all to memory, sort them, and then use 10 latest ones.

Reply:
fixed in 8ac7bb42aac85e9e420e5cfa0450d6536bfbda46